exotic/exotic_gui.py

```diff
@@ -437,14 +437,15 @@ def main():
 
                 # i, j = 1, 0
                 platesolve = tk.BooleanVar()
-                platesolve_check = tk.Checkbutton(root, text='Plate solve my images', variable=platesolve, onvalue=True,
-                                                  offvalue=False)
+                platesolve_check = tk.Checkbutton(root, text='Plate solve my first image', variable=platesolve,
+                                                  onvalue=True, offvalue=False)
                 platesolve_check.grid(row=i, column=j, sticky=tk.W, pady=2)
                 i += 1
 
                 alignment = tk.BooleanVar()
-                alignment_check = tk.Checkbutton(root, text='Align my images', variable=alignment, onvalue=1,
-                                                 offvalue=0)
+                alignment_check = tk.Checkbutton(root, text="Align my images\n(Note: Do not check if you'd like to use"
+                                                            "\nWCS in header for tracking targets)",
+                                                 variable=alignment, onvalue=1, offvalue=0, justify=tk.LEFT)
```

Comment:
What do you think about the output text? Can modify if needed.

Reply:
this might be a little confusing for users- to me, it can read as "oh hey, if we do a WCS solution with EXOTIC, then we don't need to align" - I *think* we want to say is that if someone has already done the WCS plate solution for *all* of their images prior to running EXOTIC, then the alignment is unnecessary...but, don't you check now to see if each image is already plate solved? and if so, it overrides and cancels the alignment?